src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1

```diff
@@ -584,7 +584,7 @@ function Get-Location
 {
     param([string]$providerNamespace, [string]$resourceType, [string]$preferredLocation)
     $provider = Get-AzureRmResourceProvider -ProviderNamespace $providerNamespace
-    $resourceTypes = $provider.ResourceTypes | Where-Object { $_.Name -eq $resourceType}
+    $resourceTypes = $provider.ResourceTypes | Where-Object { $_.ResourceTypeName -eq $resourceType}
```

Comment:
This change is likely breaking other tests

Reply:
I will review next week